integration-libs/opf/base/core/facade/opf-payment.service.spec.ts

```diff
@@ -39,6 +42,12 @@ class MockPaymentConnector implements Partial<OpfPaymentConnector> {
   getActiveConfigurations(): Observable<ActiveConfiguration[]> {
     return of(mockActiveConfigurations);
   }
+  getCtaScripts(
+    ctaScriptsRequest: CtaScriptsRequest
+  ): Observable<CtaScriptsResponse> {
+    console.log(ctaScriptsRequest);
```

Comment:
Leftover or intended?

Reply:
ouch, debugging leftover

Reply:
the idea is to avoid linting complaining like it was done above. I can add the command to bypass linit for this line.

Reply:
using underscore prefix seems a good alternative
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals